test/distributed/test_dynamo_distributed.py

```diff
@@ -291,13 +319,49 @@ def test_fsdp_inductor(self):
             outputs = fsdp_m(inputs)
             self.assertTrue(same(correct_outputs, outputs))
 
+
+    @skip_if_lt_x_gpu(1)
+    @unittest.skipIf(not has_triton(), "Inductor+gpu needs triton and recent GPU arch")
+    def test_fsdp_activation_checkpointing(self):
+        from torch.distributed.algorithms._checkpoint.checkpoint_wrapper import (
```

Comment:
do we want to also test the native torch.utils one too?

Reply:
Thats already tested in test/dynamo/test_activation_checkpointing.py

They are also tested heavily on HF models - more data here - https://github.com/pytorch/pytorch/pull/102935